### PR TITLE
GE21 fix nstrips for 16 eta partition option

### PR DESCRIPTION
**Pull request by watson-ij**
#### PR description:

New version of GEMSpecs which contains the right number of strips for the 16 eta partition option of GE2/1 (half compared to 8 eta partition, same number of pads as they combine strips from adjacent eta partitions). Update the test geometry to build with the new GEMSpecs version.

@bsunanda @dildick 

#### PR validation:

Checked the geometry still builds and looks correct with a modified Geometry/GEMGeometry/test/testGEMGeometry_cfg.py

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by cvuosalo** [comment]
@cmsbuild please test

File: Geometry/GEMGeometryBuilder/data/v11/GEMSpecs.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
```
Comment (cvuosalo) [inline Geometry/GEMGeometryBuilder/data/v11/GEMSpecs.xml:2]:
```suggestion
<DDDefinition>
```

Comment (cvuosalo) [reply Geometry/GEMGeometryBuilder/data/v11/GEMSpecs.xml:2]: @ianna Please confirm the long DDDefinition lines are not needed anymore.

File: Geometry/GEMGeometryBuilder/data/v11/GEMSpecsFilter.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
```
Comment (cvuosalo) [inline Geometry/GEMGeometryBuilder/data/v11/GEMSpecsFilter.xml:2]:
```suggestion
<DDDefinition>
```

File: Geometry/MuonCommonData/python/testExtendedGeometry2026D54XML_cfi.py [hunk 3]
```diff
@@ -111,8 +111,8 @@
         'Geometry/CSCGeometryBuilder/data/cscSpecsFilter.xml',
         'Geometry/CSCGeometryBuilder/data/cscSpecs.xml',
         'Geometry/RPCGeometryBuilder/data/2026/v1/RPCSpecs.xml',
-        'Geometry/GEMGeometryBuilder/data/v7/GEMSpecsFilter.xml',
-        'Geometry/GEMGeometryBuilder/data/v7/GEMSpecs.xml',
+        'Geometry/GEMGeometryBuilder/data/v11/GEMSpecsFilter.xml',
```
Comment (kpedro88) [inline Geometry/MuonCommonData/python/testExtendedGeometry2026D54XML_cfi.py:114]: Right now this change is only put in this test file. Will this be propagated to any official detector configurations?

Comment (dildick) [reply Geometry/MuonCommonData/python/testExtendedGeometry2026D54XML_cfi.py:114]: Yes, Sundanda will make a workflow for the 16-partition GE2/1 geometry. But before he can do that, I need to update the GEM-CSC trigger, EMTF and GEM validation modules to temporarily reject those 16-partition GE2/1 trigger pads until the code is updated.

File: Geometry/GEMGeometryBuilder/data/v11/GEMSpecs.xml (hunk 1 above)
Comment (watson-ij) [reply Geometry/GEMGeometryBuilder/data/v11/GEMSpecs.xml:2]: Shall I make this change?
Also, I copied and modified this from another version of the GEMSpecs, should we also update these to remove the schemas?

**Comment by bsunanda** [comment]
@cvuosalo Can you please approve this? One can clean up the header later on

File: Geometry/GEMGeometryBuilder/data/v11/GEMSpecs.xml (hunk 1 above)
Comment (cvuosalo) [reply Geometry/GEMGeometryBuilder/data/v11/GEMSpecs.xml:2]: @watson-ij I think these schemas are obsolete and no longer needed. Please change them in this PR or a later one.

**Comment by cvuosalo** [comment]
+1

File: Geometry/GEMGeometryBuilder/data/v11/GEMSpecs.xml (hunk 1 above)
Comment (watson-ij) [reply Geometry/GEMGeometryBuilder/data/v11/GEMSpecs.xml:2]: Thanks, I'll make the changes in a new PR

**Comment by kpedro88** [comment]
+upgrade

**Comment by qliphy** [comment]
+1

**Pull request merged**